common/types/offer.go

```diff
@@ -152,14 +152,18 @@ func (o *Offer) validate() error {
 	// can't be used to complete the swap if the maker does not have sufficient
 	// ETH to make the claim themselves. While relayers are not used with ERC20
 	// swaps, we still want a minimum swap amount, so we use the same value.
-	relayerFeeAsXMR, err := o.ExchangeRate.ToXMR(coins.RelayerFeeETH)
+	minAmtAsETH, err := o.ExchangeRate.ToETH(o.MinAmount)
```

Comment:
We've already validated that the exchange rate has no more than 6 decimal places and the min amount has no more than 12, so converting to ETH (a multiply) will always result in a non-rounded ETH value. The original calculation here was dividing the relayer fee by the exchange rate which can result in fractional piconeros.